kornia/augmentation/auto/operations/ops.py

```diff
@@ -0,0 +1,246 @@
+from typing import Optional, Tuple
+
+import torch
+
+from kornia.augmentation.auto.operations.base import OperationBase
+from kornia.augmentation import (
+    RandomBrightness,
+    RandomContrast,
+    RandomHue,
+    RandomSaturation,
+    RandomEqualize,
+    RandomGrayscale,
+    RandomInvert,
+    RandomPosterize,
+    RandomSolarize,
+    RandomSharpness,
+    RandomHorizontalFlip,
+    RandomVerticalFlip,
+    RandomRotation,
+)
+from kornia.core import Tensor
+from kornia.grad_estimator import STEFunction
+
+__all__ = [
+    "Brightness", "Contrast", "Hue", "Saturate", "Equalize", "Gray", "Invert", "Posterize",
+    "Solarize", "SolarizeAdd", "Sharpness", "HorizontalFlip", "VerticalFlip", "Rotate",
+]
+
+
+class Brightness(OperationBase):
```

Comment:
@shijianjian just small concern that users get confused about how to use api because of methods duplication

Reply:
It is okay since users shall not access those operations directly. They will only use the high-level APIs as `kornia.augmentation.auto.AutoAugment`. The custom policies are used as JSON than calling those operations.

Reply:
Fair point